systemd/ua-license-check.timer

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Ubuntu Advantage Timer to check for GCP Ubuntu Pro licenses
+ConditionPathExists=!/var/lib/ubuntu-advantage/marker-attached
+ConditionPathExists=/var/lib/ubuntu-advantage/marker-license-check
```

Comment:
Is there any way we can use a systemd.path file with a PathExists=/var/lib/marker-license-check to this timer instead of the ConditionPathExists in a timer definition (which I think is only calculated once at boot)?

Reply:
Yep and thats the exact problem I ran into at eod yesterday! Thanks for reminding me about the difference between the two approaches